README.md

```diff
@@ -1,10 +1,12 @@
 # Bottlerocket Update Operator
 
 The Bottlerocket update operator is a [Kubernetes operator](https://Kubernetes.io/docs/concepts/extend-Kubernetes/operator/) that coordinates Bottlerocket updates on hosts in a cluster.
+When installed, the Bottlerocket update operator starts a controller deployment in any node and agent daemon set in each Bottleorocket nodes, which takes care of periodically querying updates, drain node, and perform update when asked by controller.
```

Comment:
Reading this again, I think you may want to try rephrasing this to something similar to the example below: 


>When installed, the Bottlerocket update operator starts a controller deployment in any node and agent daemon set in each Bottleorocket nodes. This takes care of periodically querying updates, draining nodes, and performing updates when asked by the controller

Reply:
What you suggested would be incorrect information because the agent daemonset is the one that regularly queries the node. If it was not clear from the existing text, I think we should change it. What do you think of this:

> When installed, the Bottlerocket update operator starts a controller deployment in any node and agent daemon set in each Bottleorocket node. Agent daemon set takes care of periodically querying updates, draining nodes, and perform updates when asked by the controller.

Reply:
```suggestion
When installed, the Bottlerocket update operator starts a controller deployment on one node and agent daemon set on every Bottleorocket node, which takes care of periodically querying updates, draining the node, and performing an update when asked by controller.
```